### PR TITLE
Readme.md of hotelontouch #44

### LINKED ISSUE #44: Study the file structure in src and document the development guide
**Is your feature request related to a problem? Please describe.**
Edit the ```Readme.md``` in the https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/tree/main/hotelontouch but first please study the whole flow and try to run using yarn start and then study the folder structure we use and the function naming types and document the whole thing in Readme.md such that a new developer can easily understand the codebase.

### PR DESCRIPTION
**Pull request by aniketsingh98571**
Fixes #44 

Added Readme.md

- [ ]

Screenshots of the changes (If any) -

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

#### ✅️ By submitting this PR, I have verified the following

- [x] Worked on forked repo and starred the main repo
- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
1. You have given only file structures not how to setup and run the app I said preciously to give links to that also from this repository's wiki please include that
2. remove your modified package-lock.json

**Comment by aniketsingh98571** [comment]
@ayan-biswas0412 I have made the changes, please look into it.

File: hotelontouch/README.md [hunk 1]
```diff
@@ -1,70 +1,22 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `yarn build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+📦src<br>
```
Comment (ayan-biswas0412) [inline hotelontouch/README.md:1]:
```suggestion
### File Structure 
📦src<br>
```

File: hotelontouch/README.md [hunk 2]
```diff
@@ -1,70 +1,22 @@
...
+ ┃ ┗ 📜[index.js-Export your component Here too](https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/blob/main/hotelontouch/src/pages/index.js)<br>
+ ┣ 📜App.css<br>
+ ┣ 📜App.js<br>
+ ┣ 📜App.test.js<br>
+ ┣ 📜index.css<br>
+ ┣ 📜index.js<br>
+ ┣ 📜logo.svg<br>
+ ┣ 📜reportWebVitals.js<br>
+ ┗ 📜setupTests.js<br>
+ Regarding Project setup and Contribution, please refer [Wiki](https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/wiki)
```
Comment (ayan-biswas0412) [inline hotelontouch/README.md:22]:
```suggestion

 Regarding Project setup and Contribution, please refer [Wiki](https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/wiki)
```

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]

**Comment by aniketsingh98571** [comment]
@ayan-biswas0412 Committed your changes..

**Review by ayan-biswas0412 — APPROVED** [review]
nice work

**Pull request merged**